about.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>  
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;400;700&display=swap" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/5f1b7e3077.js" crossorigin="anonymous"></script>
+    <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet" type="text/css"/>
+    <link rel="stylesheet" href="style-about.css"/>
+    <script src="main.js" defer></script>
+    <title>Chef conference 2022 - about</title>
+  </head>
+
+  <body>
+    <header>
+      <nav>
+        <div class="top-bar">
+          <a><i class="fab fa-facebook-f"></i></a>
+          <a><i class="fab fa-twitter"></i></a>
+          <a>Spanish</a>
+          <a>Donations</a>
+          <a>More</a>
+        </div>
+        <ul class="mainMenu" id="img-menu-2">
+          <li><img src="./img/chef.png" alt="chef" class="chef-icon"></li>
+          <li><a href="./index.html" class="nav-1">Home</a></li>
+          <li><a href="#project" class="nav-2">History</a></li>
+          <li><a href="./about.html" class="nav-3">About</a></li>
+          <li><a href="#contact-me" class="nav-4">Reservation</a></li>
+          <li>
+            <div class="closeMenu"><i class="fa fa-times"></i></div>
+          </li>
+        </ul>
+        <div class="openMenu"><i class="fa fa-bars"></i></div>
+        <div id="overlay"></div>
+      </nav>
+      <section class="first-page">
+        <h1>Chef<br>Conference 2022</h1>
```

Comment:
- [ ] As suggested by the previous CR, to match the original design better, you should consider giving this `<h1>` some more styling. For instance, you could add a bigger font-size, bold font weight, and the provided orange color!